pep-0682.rst

```diff
@@ -159,6 +165,12 @@ the new option, %-formatting cannot.  There is already precedent for not
 extending %-formatting with new options, as was the case for the
 ``,`` option (:pep:`378`).
 
+C99 ``printf`` is already employing the ``z`` option character for another
+purpose:  qualifying the unsigned type (``u``) to match the length of
+``size_t``.  However, since the signed zero option specifically disallows
+``z`` for integer presentation types, it's possible to disambiguate the two
+uses, should C99 want to adopt this new option.
```

Comment:
This may be ignorance of how the C standard works -- I thought when standardised to a year (C 1999) the specification didn't change following that, so "should C99 want to adopt this new option" can never be true. If not please dismiss this comment.